apps/dashboard/src/pages/TestDatabase.tsx

```
@@ -0,0 +1,239 @@
+        <button
+          onClick={runTests}
+          disabled={loading}
+          className={`
+            px-6 py-3 rounded-lg font-semibold text-white transition-all
+            ${loading
+              ? 'bg-gray-600 cursor-not-allowed opacity-50'
+              : 'bg-blue-600 hover:bg-blue-700 active:scale-95'
+            }
+          `}
+        >
+          {loading ? '🔄 Running Tests...' : '▶️ Run All Tests'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)